.cbmc-batch/include/cipher_openssl.h

```diff
@@ -0,0 +1,7 @@
+#include <aws/common/byte_buf.h>
+#include <aws/cryptosdk/private/cipher.h>
+
+struct aws_cryptosdk_md_context {
+    struct aws_allocator *alloc;
+    EVP_MD_CTX *evp_md_ctx;
+};
```

Comment:
This file just copies the definition of aws_cryptosdk_md_context from cipher_openssl.c so that it can be used in the proofs. I am looking into a way to automatically generate this file during the build process of CBMC so that it doesn't have to be maintained.